rollout/replicaset.go

```diff
@@ -53,7 +53,7 @@ func (c *rolloutContext) addScaleDownDelay(rs *appsv1.ReplicaSet, scaleDownDelay
 		}
 		return nil
 	}
-	deadline := metav1.Now().Add(scaleDownDelaySeconds * time.Second).UTC().Format(time.RFC3339)
+	deadline := metav1.Now().Add(scaleDownDelaySeconds).UTC().Format(time.RFC3339)
```

Comment:
Fixed confusing method behavior where duration was expected to be supplied in nanoseconds.

Reply:
just for clarification, earlier scaleDownDelaySeconds * time.Second ==> wrong calculation? 
as scaleDownDelaySeconds is already in seconds

Reply:
All callers of this method were supplying scaleDownDelaySeconds in nanosecond duration. I also changed the callers to simply supply proper durations.